scripts/import/dbSNP_v2/load_dbsnp.pl

```diff
@@ -225,6 +276,81 @@ sub parse_refsnp {
 
 }
 
+# Flips a minor allele if there are differences in GRCh37 and GRCh38 alignment
+# Input:
+# old_minor_allele - allele from GRCh37
+# var_name         - name of the variant
+# align_diff       - alignment diff info between GRCh37 and GRCh38
+# vfs              - variation features for the variant
+# fh               - filehandle to log info and errors
+#
+# Returns:
+# new_minor_allele if flip can be done, otherwise undef
+sub flip_minor_allele {
+  my ($old_minor_allele, $var_name,
+      $align_diff, $vfs, $fh) = @_;
+
+  return if (! $old_minor_allele);
+
+  if ($align_diff->{'review_status'} ne 'ALIGN_DIFF') {
+    print $fh join("\t", $var_name, 'INFO',
+      'review_status: ' . $align_diff->{'review_status'}), "\n";
+    return;
+  }
+
+  my $grch37_loc = $align_diff->{'grch37_loc'};
+  my $grch38_loc = $align_diff->{'grch38_loc'};
+  if ($grch37_loc != $grch38_loc) {
+    print $fh join("\t", $var_name, 'SKIP',
+      "Num of GRCh37 loc ($grch37_loc) differ num of GRCh38 loc ($grch38_loc)"), "\n";
```

Comment:
Can this message be made clearer? Does this mean if there is 1 mapping on GRCh37 and 2 on GRCh38, no allele is returned but if there are 2 on both it is?

Reply:
Yes means that if the number of mappings differ between GRCh37 and GRCh38 the allele not be  reverse_complemented, no allele returned, minor allele will stay as orginal
If the number of mapping the same between GRCh37 and GRCh38 the allele will be reverse complemented -even if number of mappings both 2. Should this be restricted to 1 mapping and 1 opposite alignment for GRCh37 and GRCh38?